cypress/integration/unit_tests/polygon_draw_test.js

```diff
@@ -1,43 +1,61 @@
-import {getFirestoreRoot} from '../../../docs/firestore_document';
-import * as loading from '../../../docs/loading';
-import {initializeAndProcessUserRegions, setUpPolygonDrawing, StoredShapeData} from '../../../docs/polygon_draw';
-import * as resourceGetter from '../../../docs/resources';
-import SettablePromise from '../../../docs/settable_promise';
-import {CallbackLatch} from '../../support/callback_latch';
-import {loadScriptsBeforeForUnitTests} from '../../support/script_loader';
-import {createGoogleMap} from '../../support/test_map';
-
-const polyCoords = [
-  {lng: 1, lat: 1},
-  {lng: 1, lat: 2},
-  {lng: 13, lat: 2},
-  {lng: 13, lat: 1},
-  {lng: 1, lat: 1},
-];
-
-let polyLatLng;
-const firebaseCollection = {};
-const calculatedData = {
+import * as loading from '../../../docs/loading.js';
+import {initializeAndProcessUserRegions, setUpPolygonDrawing, StoredShapeData, transformGeoPointArrayToLatLng, userShapes} from '../../../docs/polygon_draw.js';
+import {setUserFeatureVisibility} from '../../../docs/popup.js';
+import * as resourceGetter from '../../../docs/resources.js';
+import {userRegionData} from '../../../docs/user_region_data.js';
+import {CallbackLatch} from '../../support/callback_latch.js';
+import {loadScriptsBeforeForUnitTests} from '../../support/script_loader.js';
+import {convertGoogleLatLngToObject, convertPathToLatLng, createGoogleMap} from '../../support/test_map.js';
+
+const notes = 'Sphinx of black quartz, judge my vow';
+
+const path = [{lat: 0, lng: 0}, {lat: 4, lng: 2}, {lat: 0, lng: 2}];
+
+// The helper functions in this test fall into two categories: those that do
+// their actual work during the "Cypress phase" of the test, and those that
+// execute right away. The latter type always return a non-Cypress value, and
+// should always be inside a Cypress .then() block.
+
+/**
+ * Expected data in Firestore/map popup.
+ * @typedef {{damage: number, snapFraction: number, notes: string,
+ * totalHouseholds: number}} ExpectedData
+ */
+
+const defaultData = {
   damage: 1,
   snapFraction: 0.1,
-  totalHouseholds: 3,
+  totalHouseholds: 1,
+  notes: '',
 };
+const event = new Event('overlaycomplete');
```

Comment:
I think this is only used in drawPolygon so initialize in there? Or can it be used in the Draws marker test as well?